git-publish

```diff
@@ -446,13 +459,11 @@ def menu_select(menu):
         return a
 
 def edit_email_list(cc_list):
-    tmpfile = tempfile.NamedTemporaryFile(mode='wb', suffix='.txt')
-    tmpfile.write(os.linesep.join(cc_list).encode(ENCODING))
-    tmpfile.flush()
-    edit(tmpfile.name)
+    new_content_bytes = edit_content(os.linesep.join(cc_list).encode(ENCODING), '.txt')
+    new_content = new_content_bytes.decode(ENCODING)
     r = []
-    for line in open(tmpfile.name, "r").readlines():
-        r += [x.strip() for x in line.split(",")]
+    for line in new_content.splitlines():
+        r += [x.strip() for x in line.split(",") if len(x.strip())]
```

Comment:
Why was `if len(x.strip())` added? I'm not sure if this fixes a pre-existing issue or something related to switching to edit_content().

Reply:
# Remove blank email item in list by len(x.strip())

Reply:
Why would there be ",," blank items in the comma-separated list? Is there a reason why it was necessary to add this?

Reply:
> Why would there be ",," blank items in the comma-separated list? Is there a reason why it was necessary to add this?

It's just a normalize, such as mis-typing, it's doesn't hurt anything

Reply:
I see. Please put unrelated changes into separate commits. That way it's easier to review the code and understand the reason for the changes.

Reply:
OK, split into two commits

Reply:
Thanks for your patience!